test/unit/steps/sms-notify/enter-mobile/EnterMobile.test.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const { expect } = require('test/util/chai');
+const EnterMobile = require('steps/sms-notify/enter-mobile/EnterMobile');
+const content = require('steps/sms-notify/enter-mobile/content.json');
+
+describe('EnterMobile.js', () => {
+
+    let enterMobileClass;
+
+    beforeEach(() => {
+       enterMobileClass = new EnterMobile();
+    });
+
+    afterEach(() => {
+       enterMobileClass = undefined;
+    });
+
+    describe('get url()', () => {
+
+        it('returns the correct url', () => {
```

Comment:
Can we add the url to the description please therefore adding more detail when running the tests from the command line.